envoy/config/filter/http/jwt_authn/v2/config.proto

```diff
@@ -0,0 +1,237 @@
+
+syntax = "proto3";
+
+package envoy.config.filter.http.jwt_authn.v2;
```

Comment:
What are the settlement of #458? Should this be v2alpha?

Reply:
Waiting for the decision, before that leave it "v2" for now.

Reply:
I think we have consensus there, let's make this `v2alpha`.

Reply:
Done